application/helpers/remotecontrol/remotecontrol_handle.php

```
@@ -3374,14 +3376,23 @@ public function get_uploaded_files($sSessionKey, $iSurveyID, $sToken)
             return array('status' => 'No permission');
         }
 
-        $oResponses = Response::model($iSurveyID)->findAllByAttributes(array('token' => $sToken));
+        if (empty($responseId) and empty($sToken)) {
+            return ['status' => 'Invalid arguments: both Token and Reponse ID are empty'];
+        }
+        $criteria = new CDbCriteria();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '3382', column '25'). (undefined)

(MissingImport)